css/style.css

```diff
@@ -0,0 +1,75 @@
+
+.header {
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+    max-width: 1024px;
+    width: 100%;
+    margin: 0 auto;
+    padding: 25px;
+    font-family: "Arial", sans-serif;
+    font-size: 25px;
+}
+.personal-information {
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    width: 100%;
+}
+.objective {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    margin-bottom: 10px;
+    background-color: rgb(196, 196, 196)
+}
+.avatar-container {
+    display: flex;
+    height: auto;
+}
+.avatar {
+    width: 270px;
+    border-radius: 100%;
+    margin-left: 100px;
+    margin-right: 40px;
+}
+.contact{
+    padding: 5px;
+}
+.introduction {
+    display: flex;
+    justify-content: start;
+    align-items: flex-start;
+    width: 100%;
+    max-width: 1024px;
+    margin: 0 auto;
+    border-left: rgb(194, 110, 0) solid 15px;
+    font-family: "Geneva", sans-serif;
+    font-size: 30px;
+    padding-left: 30px;
+}
+.main-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+    width: 100%;
+    max-width: 1024px;
+    margin: 0 auto;
+}
+
+.skills {
+    font-size: 30px;
+    font-family: "Geneva", sans-serif;
+    padding-right: 40px;
+}
+.education {
+    font-size: 30px;
+    font-family: "Geneva", sans-serif;
+    padding-left: 40px;
+}
```

Comment:
You can define classes like this (to remove duplications):
.skills,
.education {
font-size: 30px;
font-family: "Geneva", sans-serif;
padding-right: 40px;
}

But it would be better to create one universal css class for this case with a suitable name.

Reply:
fixed